docs/source/using/mounting-host-folders.adoc

```diff
@@ -0,0 +1,122 @@
+[[mounting-host-folders]]
+= Mounting host folders
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+[[mount-host-folders-overview]]
+== Overview
+
+Some drivers mount a host folder within the VM so that you can share files between the VM and the host.
+These folders are not currently configurable, and are different for each driver and OS that you use.
+
+[cols=",,,",options="header",]
+|========================================
+|Driver |OS |HostFolder |VM
+|Virtualbox |Linux |/home |/hosthome
+|Virtualbox |OSX |/Users |/Users
+|Virtualbox |Windows |C://Users |/c/Users
+|VMWare Fusion |OSX |/Users |/Users
+|Xhyve |OSX |/Users |/Users
+|========================================
+
+NOTE: Host folder sharing is not implemented in the KVM and Hyper-V drivers. You can
+link:#mounting-custom-shared-folders[mount a CIFS-based shared folder] inside the VM instead.
+
+[[mounting-custom-shared-folders]]
+== Mounting custom shared folders
+
+The Minishift https://github.com/minishift/minishift-b2d-iso[Boot2Docker] ISO and the
+Minishift https://github.com/minishift/minishift-centos-iso[CentOS] ISO include `cifs-utils`
+and `fuse-sshfs`. These tools allow you to mount host folders using
+https://en.wikipedia.org/wiki/Server_Message_Block[CIFS]
+or https://en.wikipedia.org/wiki/SSHFS[SSHFS].
+
+NOTE: When you use the Boot2Docker ISO along with the VirtualBox driver, VirtualBox
+guest additions are automatically enabled and occupy the _/Users_ mountpoint as
+shown in the following examples. In this case you need to use a different mountpoint.
+
+[[cifs-folder-mount]]
+=== Mounting folders with CIFS
+
+In this example, you use CIFS-based sharing to mount `C:\Users` on a Windows host
+into the Minishift VM. On Windows 10, the `C:\Users` folder is shared by default
+and only needs locally-authenticated users.
+
+.  Find the local IP address from the same network segment as your
+Minishift instance.
++
+----
+$ Get-NetIPAddress | Format-Table`
+----
+
+.  Create a mountpoint and mount the shared folder.
++
+[source,sh]
```

Comment:
> we don't need this [source, sh], I actually went and removed as many of those as I could find, it's a conversion leftover, sorry if I missed this!

-1, imo the opposite is true. We should add them were they are missing. This is the information the syntax highlighter needs to decide on the form of syntax highlighting. Why would you want to remove this? 

> Can we discuss and decide on why we need these tags and how they help us?

I am surprised that we need to discuss this, but sure:
* http://www.methods.co.nz/asciidoc/source-highlight-filter.html
* http://asciidoctor.org/docs/user-manual/#source-code-blocks

We are using embedded (source) code here. On the HTML side source rendering occurs with one of a whole list of source code highlighters (at least it can be) - CoreRay, prettify, etc. This highlighters are quite advanced and can depending on the type of source - sh, java, php, etc - apply syntax highlighting as expected per language type. However, to do this you need to provide some meta information. In this case [source,sh]. Or in case we are having a JSON example we should use [source,json]. 

Even if the highlighter does not get used, adding this information makes sense, since it adds useful meta information. 

So -1 for removing this. On the contrary, we should try adding them. This is actually what I am doing. 

On that note, longer free standing code examples and table should always also get a anchor, so that they become reference-able. Personally I like to use prefixes for this 'example-<bla>', 'table-<bla>'. If you reference these type of things from somewhere in the document it becomes clear what you are referencing even w/o looking at the reference target, e.g. <<table-addon-command-list>> makes clear that I am referencing a table even without looking at the actual target. Effectively you are creating a namespace over your anchors.